src/seedu/addressbook/AddressBook.java

```diff
@@ -394,8 +394,9 @@ private static String executeCommand(String userInputString) {
      * @return  size 2 array; first element is the command type and second element is the arguments string
      */
     private static String[] splitCommandWordAndArgs(String rawUserInput) {
-        final String[] split =  rawUserInput.trim().split("\\s+", 2);
-        return split.length == 2 ? split : new String[] { split[0] , "" }; // else case: no parameters
+    	int splittedElements = 2;
+        final String[] splittedCommand =  rawUserInput.trim().split("\\s+", splittedElements);
+        return splittedCommand.length == splittedElements ? splittedCommand : new String[] { splittedCommand[0] , "" }; // else case: no parameters
```

Comment:
For long ternary expressions, you may want to refer to the format in coding standard.